### DISCUSSION TITLE
How to configure DNSBL lookup?

### DISCUSSION THREAD

**Discussion by sol2070** [emersion/maddy#607] [Q&A] [answered]
Where do I put the configuration for DNSBL lookup?
When I add

```
check {
    dnsbl all.s5h.net b.barracudacentral.org
}
``` 
at the end of maddy.conf, I get this error on maddy restarting:
`/etc/maddy/maddy.conf:194: unknown module or global directive: check`

What am I missing?

I'm using the pre-built tarball of version 0.7.0.

Thank you!

**Comment by foxcpp** [comment, marked as answer]
`check { }` should be placed into `smtp`, `submission` or `msgpipeline` in order to specify when in particular it should be applied. You probably want to add it inside `smtp tcp://0.0.0.0:25` to apply it for inbound messages.

**Reply by sol2070** [reply]
Thank you! I put it into `submission`, as most of the attacks are targetting this. Looks like it's working!
